api/tests/certification/course/integration/infrastructure/repositories/sco-certification-candidate-repository_test.js

```diff
@@ -75,14 +101,31 @@ describe('Certification | Course | Integration | Repository | SCOCertificationCa
         organizationId: anOrganizationId,
         division: '3ème B',
       }).id;
-      const candidateId = databaseBuilder.factory.buildCertificationCandidate({
+      const candidateOfTheTargetedDivision = databaseBuilder.factory.buildCertificationCandidate({
```

Comment:
Autre proposition mais targeted me va en vrai, juste pour être iso avec le nom du test je propose given
```suggestion
      const candidateFromTheGivenDivision = databaseBuilder.factory.buildCertificationCandidate({
```

Reply:
Done